custom-demo/static/css/custom-demo.css

```diff
@@ -64,7 +64,7 @@ audio:not([controls]) {
 }
 /**
  * Address `[hidden]` styling not present in IE 8/9/10.
- * Hide the `template` element in IE 8/9/11, Safari, and Firefox < 22.
+ * Hide the `template` element in IE 8/9/10/11, Safari, and Firefox < 22.
```

Comment:
That text is from [normalize.css](https://github.com/necolas/normalize.css/blob/master/normalize.css#L74) so there's not much we can do to change it.